assets/js/listener.js

```diff
@@ -17,17 +21,48 @@ document.addEventListener('click', function(event) {
     }
 });
 
+document.getElementById("comment").addEventListener("input", function() {
+    const text = this.value.trim();
+    const sendButton = document.getElementById('send');
+    
+    if (!document.getElementById('yes').checked && !document.getElementById('no').checked) {
+        text.length > 0 ? sendButton.disabled = false : sendButton.disabled = true;
+    }
+
+    // calculate the number of characters remaining
+    counter = 350 - this.value.length;
+    document.getElementById('num-chars').innerText = counter + " characters left";
```

Comment:
One problem we would have is if a user enters a text and does a refresh, the page would reload with the text in the textfield and a disabled button and the message saying 350 chars to go.

everything being done in the listener should be moved in to a function and that function should be used for the listener and also just called when the page loads (right where the listeners are being added is fine).

Reply:
That was one of the requirements from UX: the users should be able to reload the page and get back to the initial state, where they would be able to submit feedback again.

Reply:
Implemented as suggested in the most recent commit.